controllers/components/kserve/kserve_support.go

```diff
@@ -271,3 +271,11 @@ func ownedViaFT(cli client.Client) handler.MapFunc {
 func isLegacyOwnerRef(or metav1.OwnerReference) bool {
 	return or.APIVersion == gvk.DataScienceCluster.GroupVersion().String() && or.Kind == gvk.DataScienceCluster.Kind
 }
+
+func ifGVKInstalled(kvg schema.GroupVersionKind) func(context.Context, *odhtypes.ReconciliationRequest) bool {
+	return func(ctx context.Context, rr *odhtypes.ReconciliationRequest) bool {
+		c := resources.GvkToUnstructured(gvk.CustomResourceDefinition)
+		err := rr.Client.Get(ctx, client.ObjectKey{Name: kvg.GroupKind().String()}, c)
```

Comment:
is this actually correct ?

My understanding is that for a resource of type `KnativeServing`, the call `GroupKind.String()` would produce `KnativeServing.operator.knative.dev`, however the name of the CRD is `knativeservings.operator.knative.dev`

Reply:
@lburgazzoli hmm, it seems to have the desired effect at least :thinking: Is there something that you'd prefer to have it changed to?

Reply:
That is actually the reason I've asked, so I don't know if this is the actual fix, or we are observing the desired effect as a "side effect".

```
➜ kubectl get crd KnativeServing.operator.knative.dev
Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "KnativeServing.operator.knative.dev" not found
```

I've created a PR with an utility, to properly compute the CRD name based on a GVK https://github.com/opendatahub-io/opendatahub-operator/pull/1680

Reply:
I think you're right...I just fired it up in a debugger and it does seem like it's not finding it here. I'll take a look at your PR :+1: 

However, that makes me confused in a different way: that means that this is returning an error, then this predicate is returning false to `reconciler.Dynamic`, telling it that it _shouldn't set up a watch_. So how does this "fix" the problem of the watch not being set without this predicate? Where is the side effect?

Reply:
That's a very good question, something to digg in I guess

Reply:
For now I've rebased this on top of your #1680 PR so that I can use the function from there. LMK if what you think or if you expect it to be used in a different way, thanks!